PlatformRouting/PlatformRouting/_iOS/_App/MappedUIKitRouter.swift

```diff
@@ -451,9 +451,6 @@ open class MappedUIKitRouter: MappedRouter {
                     self?.actions.removeAll(where: { (actionInList) -> Bool in
                         actionReference === actionInList
                     })
-                    if success, request.path?.hasPrefix("/action") ?? false {
-                        Tracking.shared?.view(request.path, data: request.params, from: nil, time: nil)
-                    }
```

Comment:
actions should not be view events, should be dedicated action events. Did not make scope for this PR